src/Scene/NodeProcess.js

```diff
@@ -253,48 +236,25 @@ function updateNodeImagery(scene, quadtree, node, layersConfig, force) {
             }
         }
 
-        let ancestor = null;
-
-        const currentLevel = node.materials[RendererConstant.FINAL].getColorLayerLevelById(layer.id);
-        // if this tile has no texture (level == -1), try use one from an ancestor
-        if (currentLevel === -1) {
-            ancestor = findAncestorWithValidTextureForLayer(node, l_COLOR, layer);
-        } else {
-            var targetLevel = chooseNextLevelToFetch(layer.updateStrategy.type, node.level, currentLevel, layer.updateStrategy.options);
-            if (targetLevel === currentLevel) {
-                continue;
-            }
-            if (targetLevel < node.level) {
-                ancestor = node.getNodeAtLevel(targetLevel);
-            }
-        }
-
         node.layerUpdateState[layer.id].newTry();
-
+        const searchInParent = !node.isColorLayerLoaded(layer.id) && node.parent.isColorLayerLoaded(layer.id);
         const command = {
             /* mandatory */
             layer,
             requester: node,
             priority: nodeCommandQueuePriorityFunction(node),
             earlyDropFunction: refinementCommandCancellationFn,
-            /* specific params */
-            ancestor,
-            /* redraw only if we're aren't using a texture from our parent */
-            redraw: (ancestor == null),
+            parentTextures: searchInParent ? node.parent.getLayerTextures(l_COLOR, layer.id) : undefined,
         };
 
         promises.push(quadtree.scheduler.execute(command).then(
             (result) => {
-                const level = ancestor ? ancestor.level : node.level;
-                // Assign .level to texture
                 if (Array.isArray(result)) {
-                    for (let j = 0; j < result.length; j++) {
-                        result[j].texture.level = level;
-                    }
-
                     node.setTexturesLayer(result, l_COLOR, layer.id);
                 } else if (result.texture) {
-                    result.texture.level = level;
+                    if (!result.texture.coordWMTS) {
+                        result.texture.coordWMTS = node.wmtsCoords[layer.options.tileMatrixSet][0];
```

Comment:
[except for WMS, see comment](https://github.com/iTowns/itowns2/pull/244#discussion_r104423818)